src/template.c

```diff
@@ -5815,13 +5821,17 @@ void TemplateInstance::semantic(Scope *sc, Expressions *fargs)
     Dsymbols *target_symbol_list;
     size_t target_symbol_list_idx = 0;
     //if (sc->scopesym) printf("3: sc is %s %s\n", sc->scopesym->kind(), sc->scopesym->toChars());
-    if (!tinst && sc->scopesym && sc->scopesym->members)
+    if (0 && !tinst && sc->scopesym && sc->scopesym->members)
```

Comment:
Yes. By this change, all template instances are inserted to root module members, and their semantic3 will get invoked.